### ISSUE TITLE
onnx_INTERPRETER.model_convtranspose_output_shape prints extra info

### ISSUE THREAD

**Issue by jnorwood** [NervanaSystems/ngraph#3214]
open onnx_INTERPRETER.model_convtranspose_output_shape dumps 160 lines of data into the  make check results.  Probably unintended, since other tests don't do this during make check execution. 

```
[ RUN      ] onnx_INTERPRETER.model_convtranspose_output_shape
First 160 results
   0 ref: 0                 actual: 0               
1    ref: 0                 actual: 0               
2    ref: 1                 actual: 1               
3    ref: 1                 actual: 1               
4    ref: 3                 actual: 3          
```

**Comment by tsocha** [comment]
Thanks for pointing it out.
#3217 would fix this problem

**Comment by jnorwood** [comment]
verified fixed by #3217
